dexterity/membrane/behavior/password.py

```diff
@@ -22,6 +22,23 @@
 from zope.interface import provider
 
 
+class BCRYPTEncryptionScheme(object):
+    """A BCRYPT AuthEncoding."""
+
+    def encrypt(self, pw):
+        return bcrypt.hashpw(pw, bcrypt.gensalt())
+
+    def validate(self, reference, attempt):
+        try:
+            valid = bcrypt.hashpw(attempt, reference) == reference
+        except ValueError:
+            valid = False
+        return valid
+
+
+AuthEncoding.registerScheme('BCRYPT', BCRYPTEncryptionScheme())
```

Comment:
I've just forked AccessControl, and was about to try and this there.
It fails one test: the long password test.

TIL that bcrypt passwords need to be of length <= 50:
http://security.stackexchange.com/questions/39849/does-bcrypt-have-a-maximum-password-length
As such, I think it would make sense for the dx.membrane to implement `IPasswordChecker` check accordingly in dexterity.membrane.

Also, it seems the docstring/implementation for that interface is slightly off:

https://github.com/netsight/dexterity.membrane/blob/master/dexterity/membrane/behavior/password.py#L69-L74

I'll was about to submit a pull request (https://github.com/zopefoundation/AccessControl/compare/master...netsight:master)
but I think there may be some resistance, especially given tres's comments here:
https://github.com/zopefoundation/AccessControl/pull/10

I'll submit it anyway, and if its not possible, I gues it could live here for now,
or else do what PlonePAS does to Products.PluggableAuthService.
Perhaps add it to Products.CMFPlone ? I do not know the best place should AccessControl itself 
not be possible.

(We are were hoping to get this into ploneintranet 1.0 before the release)